docs/source/UserGuide/GettingStarted/InstallingClient/index.rst

```diff
@@ -25,19 +25,19 @@ Download the *dirac-install* script from::
 
 Choose the directory where you want to install the DIRAC software and run the dirac-install script from
 this directory giving the appropriate version of the DIRAC release,
-and, the version of the "lcgBundle" (with "-g" option) that you want to use::
+and the version of the "lcgBundle" (with "-g" option) that you want to use::
 
   dirac-install -r v6r20p14 -g v14r2
 
-The example above assumes that you need version v6r20p14, and that with it you are installing lcgBundle version v14r2.
+The example above assumes that you need the release version v6r20p14 and along with that lcgBundle version v14r2 to be installed.
 
-An "lcgBundle" is simply a tarball containing a number of statically-compiled libraries that are used
+An "lcgBundle" is simply a tarball(.tar) file containing a number of statically-compiled libraries that are used
 for interacting with grid environments (e.g. GFAL2, or ARC, or Condor).
-The libraries in a "lcgBundle" are not maintained within DIRAC, but DIRAC may use them.
-The produced lcgBundles can be found in `this server <http://diracproject.web.cern.ch/diracproject/lcgBundles/>`_.
+The libraries in "lcgBundle" are not maintained within DIRAC, but DIRAC may use them.
+The produced lcgBundles can also be found on `this server <http://diracproject.web.cern.ch/diracproject/lcgBundles/>`_.
 
 
-This installs the software and you should get the following directories and files created::
+This installs the software as well as creating the following directories and files::
```

Comment:
Well the software is installed in these directories, so the "as well as" is not really correct.